robot_sdk/SDK/owl_ec_sdk/robot.py

```
@@ -145,13 +147,54 @@ def calculate_inverse_kinematics(self, target_pose):
+        if not self.connect_success:   
+            return None  # Connection failed
+        
+        try:
+            suc , result , id=self.sendCMD("convertPoseFromUserToCart",{"targetPose":target , "userNo":user_num, "unit_type":1 })
+            if suc:
+                return result
+            else:
+                print("User to base coordinate conversion failed:", result)
+                return None
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)